learningequality/studio
# Rewriting migration history

## Description

Makes complete a null boolean field
Sets complete to True explicitly in places where ContentNodes are created not on the frontend
Rewrites migration history to prevent setting a default value to complete
Changes the mark incomplete command to filter by null complete values, so it can be run repeatedly
Rewrites migration history to prevent creating, dropping, and recreating an index due to a typo during development

## Steps to Test

On current develop, migrate back to migration 0115

On this branch, migrate all the way forward

On this branch, migrate back to migration 0115

On current develop migrate all the way forward

On this branch migrate all the way forwawrd

Check that none of these operations causes an error

Run mark_incomplete management command
